internal/cri/config/config.go

```diff
@@ -539,6 +542,10 @@ func ValidateRuntimeConfig(ctx context.Context, c *RuntimeConfig) ([]deprecation
 			c.ContainerdConfig.Runtimes[k] = r
 		}
 
+		if len(r.Snapshotter) == 0 {
+			r.Snapshotter = Snapshotter
```

Comment:
let's leave the default nil here and call the RuntimeSnapshotter func wherever the snapshotter is needed:  https://github.com/containerd/containerd/blob/main/internal/cri/server/images/service.go#L160
see comment below